source/includes/reference/_app_webhooks.md

```diff
@@ -0,0 +1,209 @@
+# Application Webhooks
+
+Application webhook subscription is a mechanism that will allow you to receive notifications to your servers whenever various events happen in relation to different resources created by an ***application***. You can subscribe to receive webhooks about resource created under a specific [***profile***](#profile-webhooks).
+
+Before proceeding, make sure the endpoint where you intend to receive webhooks satisfies the following requirements:
+
+* Has a valid domain name (IPs are disallowed)
+* Listens to HTTPS requests on port `443`
+* Has a valid HTTPS certificate signed by a trusted Certificate Authority - CA (self-signed or expired certificates are not accepted)
+* Does not include any query arguments in the URL
+
+`https://webhooks.example.com/balance-change` is a valid URL; `http://webhooks.example.com:8080/hook.php?type=balance` is not.
+
+You can have multiple subscriptions per event type though be mindful you will receive duplicate callbacks, one for each subscription.
+
+> **NB! Please note that you have to use client level token for making following requests.**
+
+Find out more about webhook events [here](#webhook-events).
+
+## Create
+
+
+> Example Request:
+
+```shell
+curl -X POST "https://api.transferwise.com/v3/applications/{clientKey}/subscriptions" \
+  -H "Authorization: Bearer <your client level token>" \
+  -H "Content-Type: application/json" \
+  -d '{
+       "name": "Webhook Subscription #1",
+       "trigger_on": "transfers#state-change",
+       "delivery": {
+         "version": "2.0.0",
+         "url": "https://your.webhook.url/12345"
+         }
+      }'
+```
+
+> Example Response:
+
+```json
+{
+    "id": "72195556-e5cb-495e-a010-b37a4f2a3043", 
+    "name": "Webhook Subscription #1",
+    "delivery": {
+        "version": "2.0.0",
+        "url": "https://your.webhook.url/12345" 
+    },
+    "trigger_on": "transfers#state-change", 
+    "scope": {
+        "domain": "application", 
+        "id": "<your client key>"
+    },
+    "created_by": {
+        "type": "application",
+        "id": "<your client id>", //clientId and key are not always the same
+    },
+    "created_at": "2008-09-15T15:53:00+00:00Z",
```

Comment:
Not sure it matters, but that timestamp does not look correct.  Either `+00:00` or `Z`, pick one?